src/apps/monitoring/kadiska/plugin.pm

```diff
@@ -30,11 +30,13 @@ sub new {
     bless $self, $class;
 
     $self->{modes} = {
-        'list-watchers'         => 'apps::monitoring::kadiska::mode::listwatchers',
+        'alerts'                => 'apps::monitoring::kadiska::mode::alerts',
+        'list-alertrules'       => 'apps::monitoring::kadiska::mode::listalertrules',
```

Comment:
it should be better to use the name ```list-alerts```

Reply:
I don't see it that way. This mode is about listing alert rules in order to create services that will count the number of alerts by alert rule.

Reply:
so call it list-alert-rules